gags.js

```diff
@@ -244,8 +251,10 @@ function findCombo({
   game = 'ttr',
   stunTrack = null
 }) {
-  if (numToons === 1 && gagTrack === 'drop')
-    throw new Error('Invalid arguments: a drop combo must have 2 or more toons (for a stun gag)');
+  if (numToons === 1 && gagTrack === 'drop') {
+    // For drop combos with one toon, add two skips
+    numToons += 2;
```

Comment:
I'm a little confused on what this does? Since you included this:
```js
if (!(gagTrack === 'drop' && numToons === 1)) {
  arr.push(Combo({ numToons, cogLvl, gagTrack, stunTrack }));
}
```
it looks like we're still not calling `findCombo` with `gagTrack='drop'` and `numToons=1` specified.

Alternatively, for single toon drop combos, we could forego the stun gag, but I'm fine with the current solution of not displaying single toon drop combos.

Reply:
True, that's actually just an artifact, it was the first thing I changed in the code when I was still figuring out how everything was connected, I've undone this modification as it is pointless now due to upstream changes.